### PR TITLE
Address tutorial points of confusion

### PR DESCRIPTION
**Pull request by jwmueller**
Adds pointer to Colab GPU runtime for the relevant tutorials, before packages are installed.

Adds instructions that user's own dataset classes must be represented as integer indices 0,1,...,K.

Thanks to @calebchiam for pointing these out!

### CODE REVIEW

File: docs/source/tutorials/audio.ipynb [hunk 1]
```diff
@@ -157,7 +157,7 @@
     "id": "tRvNnyB0e_IE"
    },
    "source": [
-    "The audio data are .wav files in the `recordings/` folder. Note that the label for each audio clip (i.e. digit from 0 to 9) is indicated in the prefix of the file name (e.g. `6_nicolas_32.wav` has the label 6).\n"
+    "The audio data are .wav files in the `recordings/` folder. Note that the label for each audio clip (i.e. digit from 0 to 9) is indicated in the prefix of the file name (e.g. `6_nicolas_32.wav` has the label 6). If instead applying cleanlab to your own dataset, its classes should be represented as integer indices 0,1,...,K."
```
Comment (anishathalye) [inline docs/source/tutorials/audio.ipynb:160]: `K-1`?

File: docs/source/tutorials/audio.ipynb [hunk 2]
```diff
@@ -327,7 +345,7 @@
     "# Create dataframe with .wav file names\n",
     "df = pd.DataFrame(file_paths, columns=[\"wav_audio_file_path\"])\n",
     "df[\"label\"] = df.wav_audio_file_path.map(lambda x: int(Path(x).parts[-1].split(\"_\")[0]))\n",
-    "\n",
+    "# Note: Classes must be represented as integer indices 0,1,...,K\n",
```
Comment (anishathalye) [inline docs/source/tutorials/audio.ipynb:348]: `K-1`?

File: docs/source/tutorials/image.ipynb [hunk 3]
```diff
@@ -132,6 +134,8 @@
     "\n",
     "Assign your data's features to variable `X` and its labels to variable `y` instead.\n",
     "\n",
+    "Your classes (and entries of `y`) should be represented as integer indices 0,1,...,K.\n",
```
Comment (anishathalye) [inline docs/source/tutorials/image.ipynb:137]: `K-1`?

File: docs/source/tutorials/tabular.ipynb [hunk 4]
```diff
@@ -152,6 +152,8 @@
     "Bringing Your Own Data (BYOD)?\n",
     "\n",
     "You can easily replace the above with your own tabular dataset, and continue with the rest of the tutorial.\n",
+    " \n",
+    "Your classes (and entries of `y`) should be represented as integer indices 0,1,...,K.   \n",
```
Comment (anishathalye) [inline docs/source/tutorials/tabular.ipynb:156]: `K-1`?

File: docs/source/tutorials/text.ipynb [hunk 5]
```diff
@@ -170,6 +172,8 @@
     "\n",
     "You can easily replace the above with your own text dataset, and continue with the rest of the tutorial.\n",
     "\n",
+    "Your classes (and entries of `full_labels`) should be represented as integer indices 0,1,...,K.\n",
```
Comment (anishathalye) [inline docs/source/tutorials/text.ipynb:175]: `K-1`?

**Review by anishathalye — COMMENTED** [review]
lgtm, but s/K/K-1/ ?

File: docs/source/tutorials/audio.ipynb (hunk 2 above)
Comment (cgnorthcutt) [reply docs/source/tutorials/audio.ipynb:348]: use `num_classes - 1`

File: docs/source/tutorials/image.ipynb (hunk 3 above)
Comment (cgnorthcutt) [reply docs/source/tutorials/image.ipynb:137]: `num_classes - 1`

File: docs/source/tutorials/tabular.ipynb (hunk 4 above)
Comment (cgnorthcutt) [reply docs/source/tutorials/tabular.ipynb:156]: see above

File: docs/source/tutorials/text.ipynb (hunk 5 above)
Comment (cgnorthcutt) [reply docs/source/tutorials/text.ipynb:175]: `num_casses - 1`

**Review by cgnorthcutt — APPROVED** [review]

**Review by cgnorthcutt — APPROVED** [review]
LGTM.

**Pull request merged**